source/references/UI_spec.rst

```diff
@@ -571,4 +633,7 @@ Each field in the ``parameters`` section can have the following properties:
 
    <a href="https://github.com/kbaseapps/fba_tools/blob/master/ui/narrative/methods/build_metabolic_model/display.yaml" target="_blank">Example</a>
 
+.. |sample_uploader_link| raw:: html
+    <a href="https://github.com/kbaseapps/sample_uploader/blob/master/ui/narrative/methods/filter_samplesets/spec.json#L70" target="_blank">Sample Uploader app</a>
```

Comment:
I think this isn't correctly rendering in the HTML view

Reply:
Its showing vertical bars around the link

Reply:
whoops, missed a line break. Should be fixed now.

Reply:
Can you link to a specific commit in case that file changes?

Reply:
The other links should probably be updated too.

Reply:
Done, should be good now